pkg/bootstrap/config.go

```diff
@@ -283,6 +283,7 @@ func extractRuntimeFlags(cfg *model.NodeMetaProxyConfig) map[string]string {
 		"envoy.reloadable_features.require_strict_1xx_and_204_response_headers":                                "false",
 		"re2.max_program_size.error_level":                                                                     "32768",
 		"envoy.reloadable_features.http_reject_path_with_fragment":                                             "false",
+		"envoy.reloadable_features.no_extension_lookup_by_name":                                                "true",
```

Comment:
override to false as true is default.

Reply:
for each reloadable feature, there is an expiration date, so there needs to be some follow up work once the flag is gone.

Reply:
Updated.

Reply:
I'm not sure what to do about the expiration date. Is that something we should do now, knowing that we understand what reloadable feature is the causing the failure? Then maybe this PR isn't really relevant at this point.

Reply:
I don't want to hold update now, but we probably need an issue, and reference it here.

Reply:
Looks like some unit tests need the same change applied because they use random bootstrap.